tools/migration/db/util/mysql_pgsql_data_converter.py

```diff
@@ -148,6 +149,12 @@ def write_sequence(pgsql_dump, table_name, table_columnn):
     pgsql_dump.write("SELECT setval('%s_%s_seq', max(%s)) FROM %s;\n" % (table_name, table_columnn, table_columnn, table_name))
     pgsql_dump.write("ALTER TABLE \"%s\" ALTER COLUMN \"%s\" SET DEFAULT nextval('%s_%s_seq');\n" % (table_name, table_columnn, table_name, table_columnn))
 
+def alter_sequence(pgsql_dump, table_name, table_columnn):
```

Comment:
In pgSQL 9.5+ there's support in `CREATE SEQUENCE IF NOT EXIST` 

https://www.postgresql.org/docs/current/sql-createsequence.html